sections/builder.js

```diff
@@ -1526,28 +1526,40 @@ export default [
   {
     type: "method",
     method: "offset",
-    example: ".offset(value)",
-    description: "Adds an offset clause to the query.",
+    example: ".offset(value, options={skipBinding: boolean})",
+    description: "Adds an offset clause to the query. An optional skipBinding parameter may be specified to insert the raw value in the query.",
```

Comment:
Description sounds somewhat confusing outside of context. Can you provide more details here with regards to what `insert the raw value` means in this context, or give an example where that would be relevant?

Reply:
I rework description and example. Feel free to suggest me something if it's not good, as you can see I am not very good in english :p